migrations/sqls/20190806200938-issue-202-down.sql

```diff
@@ -0,0 +1,21 @@
+/* Replace with your SQL commands */
+UPDATE services
+SET programId = (
+  SELECT id FROM programs WHERE programName = "Financial Education / Coaching"
+)
+WHERE serviceName = "Leadership Development - Monthly Meetings";
+
+DELETE FROM services
+WHERE serviceName = "General Consultation"
+OR serviceName = "Translation"
+OR serviceName = "Green Card Renewal"
+OR serviceName = "I-821 Temporary Protected Status (TPS)"
+OR serviceName = "I-765 Application for Employment Authorization"
+OR serviceName = "Freedom of Information Act (FOIA)"
+OR serviceName = "Status Check"
+OR serviceName = "I-94 Request"
+OR serviceName = "Family Petition - Consular Processing";
+
+UPDATE services
+SET serviceName = "Family Petition"
+WHERE serviceName = "Family Petition - Adjustment of Status";
```

Comment:
The "no new line at end of file" indicator popped up here.

Reply:
That's okay -- new lines at end of file are something that prettier adds to most files, but they aren't really important or even necessarily better. I guess prettier doesn't currently support formatting SQL files, which is why it didn't add the new line to this one.